### DISCUSSION TITLE
How to set the app or version?

### DISCUSSION THREAD

**Discussion by mingzaily** [kreait/firebase-php#549] [Q&A]
![image](https://user-images.githubusercontent.com/44798713/105454218-76b34800-5cbc-11eb-87b3-49a207b88167.png)
This is my code.
![image](https://user-images.githubusercontent.com/44798713/105454275-8c287200-5cbc-11eb-9227-53e8e615d6c2.png)
What should I do? Thank you.

**Comment by jeromegamez** [comment]
Unfortunately, there's no way built-into the FCM REST API (which this and the other SDKs use) to target a specific app or app version. (See [this StackOverflow answer by one of Firebase's own engineers](https://stackoverflow.com/a/54446622/284325), Frank van Puffelen). If you want to target an App and/or Version, I think you need to send your messages via the Firebase Console…
